testsuite/conformance-test-plugins/azure-arc-agent-cleanup.yaml

```diff
@@ -3,7 +3,7 @@ sonobuoy-config:
   plugin-name: azure-arc-agent-cleanup
```

Comment:
This change is already present in the target branch. You can remove this change

Reply:
taken care